integrations-and-supported-tools/pytorch-ignite/scripts/Neptune_Pytorch_Ignite.py

```diff
@@ -0,0 +1,188 @@
+import torch
+import torch.nn.functional as F
+from ignite.contrib.handlers.neptune_logger import (
+    GradsScalarHandler,
+    NeptuneLogger,
+    NeptuneSaver,
+    WeightsScalarHandler,
+    global_step_from_engine,
+)
+from ignite.engine import Events, create_supervised_evaluator, create_supervised_trainer
+from ignite.handlers import Checkpoint
+from ignite.metrics import Accuracy, Loss
+from ignite.utils import setup_logger
+from torch import nn
+from torch.optim import SGD
+from torch.utils.data import DataLoader
+from torchvision.datasets import MNIST
+from torchvision.transforms import Compose, Normalize, ToTensor
+
+# Define hyper-parameters
+params = {
+    "train_batch_size": 64,
+    "val_batch_size": 64,
+    "epochs": 10,
+    "lr": 0.01,
+    "momentum": 0.5,
+}
+
+
+# Create model
+class Net(nn.Module):
+    def __init__(self):
+        super(Net, self).__init__()
+        self.conv1 = nn.Conv2d(1, 10, kernel_size=5)
+        self.conv2 = nn.Conv2d(10, 20, kernel_size=5)
+        self.conv2_drop = nn.Dropout2d()
+        self.fc1 = nn.Linear(320, 50)
+        self.fc2 = nn.Linear(50, 10)
+
+    def forward(self, x):
+        x = F.relu(F.max_pool2d(self.conv1(x), 2))
+        x = F.relu(F.max_pool2d(self.conv2_drop(self.conv2(x)), 2))
+        x = x.view(-1, 320)
+        x = F.relu(self.fc1(x))
+        x = F.dropout(x, training=self.training)
+        x = self.fc2(x)
+        return F.log_softmax(x, dim=-1)
+
+
+model = Net()
+device = "cuda" if torch.cuda.is_available() else "cpu"
+model.to(device)  # Move model before creating optimizer
+
+
+# Define DataLoader()
+def get_data_loaders(train_batch_size, val_batch_size):
+    data_transform = Compose([ToTensor(), Normalize((0.1307,), (0.3081,))])
+
+    train_loader = DataLoader(
+        MNIST(download=True, root=".", transform=data_transform, train=True),
+        batch_size=train_batch_size,
+        shuffle=True,
+    )
+
+    val_loader = DataLoader(
+        MNIST(download=False, root=".", transform=data_transform, train=False),
+        batch_size=val_batch_size,
+        shuffle=False,
+    )
+    return train_loader, val_loader
+
+
+train_loader, val_loader = get_data_loaders(params["train_batch_size"], params["val_batch_size"])
+
+# Create optimizer, trainer, and logger
+optimizer = SGD(model.parameters(), lr=params["lr"], momentum=params["momentum"])
+criterion = nn.CrossEntropyLoss()
+
+trainer = create_supervised_trainer(model, optimizer, criterion, device=device)
+
+# (Neptune) Create NeptuneLogger()
+neptune_logger = NeptuneLogger(
+    api_token="ANONYMOUS",
```

Comment:
Whenever PR is ready, we should update this to `neptune.ANONYMOUS_API_TOKEN`.

Reply:
@normandy7 - is there any preference between using `neptune.ANONYMOUS_API_TOKEN` or `"ANONYMOUS"`? I prefer the later as it can be better incorporated within CI/CD scripts

Reply:
Just to document the reason I think `neptune.ANONYMOUS_API_TOKEN` is better

* If in future, "ANONYMOUS" is changed/dropped then we might have come back and update
* If I search `anonymous` in docs (https://docs.neptune.ai/?q=ANONYMOUS), then almost all the top hits correspond to `neptune.ANONYMOUS_API_TOKEN`.

Reply:
Wasn't this to be changed to `neptune.ANONYMOUS_API_TOKEN`?